README.md

```diff
@@ -46,10 +46,10 @@ in-memory format. [Python Bindings](https://github.com/apache/datafusion-python)
 
 Here are links to some important information
 
-- [Project Site](https://arrow.apache.org/datafusion)
-- [Installation](https://arrow.apache.org/datafusion/user-guide/cli/installation.html)
-- [Rust Getting Started](https://arrow.apache.org/datafusion/user-guide/example-usage.html)
-- [Rust DataFrame API](https://arrow.apache.org/datafusion/user-guide/dataframe.html)
+- [Project Site](https://datafusion.apache.org/)
+- [Installation](https://datafusion.apache.org/user-guide/cli/installation.html)
+- [Rust Getting Started](https://datafusion.apache.org/user-guide/example-usage.html)
+- [Rust DataFrame API](https://datafusion.apache.org/user-guide/dataframe.html)
 - [Rust API docs](https://docs.rs/datafusion/latest/datafusion)
 - [Rust Examples](https://github.com/apache/datafusion/tree/master/datafusion-examples)
 - [Python DataFrame API](https://arrow.apache.org/datafusion-python/)
```

Comment:
What should the new URL be for https://arrow.apache.org/datafusion-python/ ? I'm thinking https://datafusion.apache.org/python. That change will need to be made in github.com/apache/datafusion-python first

Reply:
It will also need a redirect link similar to https://github.com/apache/arrow-site/pull/502

Reply:
Related change to datafusion-python: https://github.com/apache/datafusion-python/pull/643